bin/consul-templaterb

```diff
@@ -101,8 +101,8 @@ optparse = OptionParser.new do |opts|
     options[:vault][:token] = vault_token
   end
 
-  options[:vault][:token_renew] = TRUE
-  opts.on('', '--vault-renew [boolean]', Boolean, 'Disable auto-renewal of the Vault token') do |vault_renew|
+  options[:vault][:token_renew] = true
+  opts.on('', '--[no-]vault-renew', 'Control auto-renewal of the Vault token. Default: activated') do |vault_renew|
```

Comment:
No clear, Default: false => deactivated would be evaluate as true right ?
Do we want to set it to false ? To me, it make more sense to defaults to true, what do you think?

Reply:
It's a ruby sugar to add the [no-] modifier to boolean flags which is optional and pass the variable in block as false instead of true

with a default to true and this kind of flag, you would have to use --no-vault-renew